client/src/components/teamPage/teamPage.css

```diff
@@ -5,3 +5,7 @@
   margin-bottom: 40px;
   margin-top: 20px;
 }
+
+.placeholder-text::placeholder {
+  color: #A9A9A9
+}
```

Comment:
Ditto, this doesn't seem to be used.

Reply:
See comment above https://github.com/Researchify/Researchify/pull/150#discussion_r686726239